docs/developers/components/managing.md

```diff
@@ -0,0 +1,97 @@
+
+> Before continuing, ensure Harper is installed and the `harperdb` CLI is available. For more information, review the [installation guide](../../deployments/install-harperdb/README.md).
+
+When developing a component locally there are a number of ways to run it on Harper.
+
+The quickest way to run a component is by using the `dev` or `run` commands (i.e. `harperdb run .`) within the component directory. Stop execution by sending a SIGINT (generally CTRL/CMD+C) signal to the process.
+
+Alternatively, to mimic interfacing with a hosted Harper instance, use operation commands instead.
+
+1. Start up Harper with `harperdb start`
+2. "Deploy" the component to the local instance by executing `harperdb deploy_component project=<name>` from within the component directory
+   1. Make sure to omit the `target` option so that it _deploys_ to the Harper instance running locally
+3. Depending on the needs of the component
+   1. If it needs a main thread restart, completely stop and start Harper again with `harperdb stop` and `harperdb start`
+   2. If it only needs a worker thread restart, run `harperdb restart` (or include `restart=true` in the deploy command from step 2)
+4. If changes need to be made to the component run `harperdb deploy_component project=<name>` again from within the component directory. It will automatically apply any changes.
+5. To remove the component use `harperdb drop_component project=<name>`
+
+> Not all [component operations](../operations-api/components.md) are available via CLI. When in doubt, switch to using the Operations API via network requests to the local Harper instance.
+
+## Hosted Harper Instance
+
+TODO
+
+## Advanced
+
+The previous methodologies should be sufficient for most developers, but a more advanced process does exist. The following methods should be executed with caution as they can have unintended side-effects. Always backup any critical Harper instances before continuing.
+
+First, locate the Harper installation **rootPath** directory. Generally, this is `/hdb`. It can be retrieved by running `harperdb get_configuration` and looking for the **rootPath** field.
+
+> For a useful shortcut on UNIX machines run: `harperdb get_configuration json=true | jq ".rootPath" | sed 's/"//g'`
+
+This path is the Harper instance. Within this directory, locate the root config titled **harperdb-config.yaml**, and the components root path. The components root path will be `<rootPath>/components` by default, but it can also be configured. If necessary, use `harperdb get_configuration` again and look for the **componentsRoot** field for the exact path.
+
+### Adding components to root
+
+Similar to how components can specify other components within their **config.yaml**, components can be added to Harper by adding them to the **harperdb-config.yaml**.
```

Comment:
Inline code instead of bold?

```suggestion
Similar to how components can specify other components within their `config.yaml`, components can be added to Harper by adding them to the `harperdb-config.yaml`.
```